core/entrypoint/docker-entrypoint.sh

```diff
@@ -40,5 +42,31 @@ else
     echo "$0: No files found in /docker-entrypoint.d/, skipping configuration"
 fi
 
+# Add DISPLAY to APP_CMD_PREFIX
+if [ -n "$APP_CMD_PREFIX" ]; then
+    APP_CMD_PREFIX="export DISPLAY=$DISPLAY;$APP_CMD_PREFIX"
+else
+    APP_CMD_PREFIX="export DISPLAY=$DISPLAY"
+fi
+
+# Ensure CARD and APP_CMD are set
+: "${CARD:?Environment variable CARD is required but not set}"
```

Comment:
Can we defaults `CARD` to `none`?

Reply:
![image](https://github.com/user-attachments/assets/aceb20fe-32b7-4ca5-b0fc-28507db00da3)